.github/workflows/wss-scan.yml

```diff
@@ -0,0 +1,46 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 3 * * *'
+  repository_dispatch:
+    types: [wss-scan]
+#  push:
+#    branches: [sreekarj_ws]
+
+jobs:
+  whitesource-vulnerability-scan:
+    name: WhiteSource Vulnerability Scan
+    runs-on: ubuntu-18.04
+    timeout-minutes: 80
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
I am not familiar with white source, but does white source have to use java 14?
what is their supported java version on their official documentation?

Reply:
Official doc: https://whitesource.atlassian.net/wiki/spaces/WD/pages/804814917/Unified+Agent+Configuration+File+and+Parameters

Supported versions : 
Java JDK 8
Java JRE 8
Java JDK 11

Reply:
Then probably we can change to JDK8 or JDK11.

Reply:
The tests ran successfully for JDK 14, so I have included that.

Reply:
The supported docs does not list JDK14, even though bc might works, still not recommend it.
Run successful doesnt mean we should just use it.